src/components/ResourceCard/AddResource/ResourceDescribeActivity.js

```diff
@@ -32,15 +32,10 @@ export const uploadThumb = async (e, props) => {
   const formData = new FormData();
   try {
     formData.append('thumb', e.target.files[0]);
-
     imageValidation = '';
     await props.uploadResourceThumbnailAction(formData);
   } catch (err) {
-    Swal.fire({
-      icon: 'error',
-      title: 'Error',
-      text: 'Failed to upload thumb.',
-    });
+    console.log(err);
```

Comment:
Hmm... don't we show any error message here in case it is failing upload on server?

Reply:
it is being taken care from  actions/services